recipes/kaolin/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "kaolin" %}
+{% set version = "0.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIAGameWorks/kaolin/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7fa6b794d83fca9b4a1eb93b13ead60767b42d5a8a25162ee73be3cead1a8c8c
+
+build:
+  number: 0
+  skip: true  # [cuda_compiler_version == "None" or cuda_compiler_version == "10.2"]
+
+requirements:
+  build:
+    - cython
+    - {{ compiler('cuda') }}
+    - {{ compiler('cxx') }}
+    - python
+  host:
+    - pip
+    - python
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - pillow
+    - tqdm
+    - pytorch
+   
+test:
+  imports:
+    - kaolin
+
+about:
+  home: https://developer.nvidia.com/nvidia-kaolin 
+  license: Apache-2.0 
+  license_family: APACHE
+  license_file: LICENSE
+  summary: 'A Pytorch Library for Accelerating 3D Deep Learning Research '
+  description:  'NVIDIA Kaolin library provides a PyTorch API for working with a variety of 3D representations and includes a growing collection of GPU-optimized operations such as modular differentiable rendering, fast conversions between representations, data loading, 3D checkpoints and more.'
```

Sorry missed this before

Would suggest wrapping this to improve readability (for [example]( https://github.com/conda-forge/staged-recipes/blob/04278118ba85d13892e0df8c92a7dc5111cae2f9/recipes/example/meta.yaml#L76-L80 ))

```suggestion
  description: |
      NVIDIA Kaolin library provides a PyTorch API for working with a variety
      of 3D representations and includes a growing collection of
      GPU-optimized operations such as modular differentiable rendering,
      fast conversions between representations, data loading,
      3D checkpoints and more.
```